Add Current Module 9227

oscarfonloz opened the pull request:
# :warning: [PR 137](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/pull/137) must be merged first into main.

- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/blob/main/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adds scripting support for the current module 9227 (4 channel Arms)

### Why should this Pull Request be merged?

Adds a new module

### What testing has been done?

Added unit tests for this specific module and ensured other tests passed

rtzoeller commented:
- [ ] I'm having trouble finding meaningful differences in the following VIs. Can you tell me what changed (or if something needs to be changed), or revert them if they should not have changed?
 - `Convert Thermocouple Module to Slot Configuration.vi`
 - `Set Slot.vi`

Otherwise the changes look good.

oscarfonloz commented:
> * [x]  I'm having trouble finding meaningful differences in the following VIs. Can you tell me what changed (or if something needs to be changed), or revert them if they should not have changed?
> * `Convert Thermocouple Module to Slot Configuration.vi`
> * `Set Slot.vi`
> 
> Otherwise the changes look good.

I removed `Convert Thermocouple Module to Slot Configuration.vi` from the PR
Set Slot.vi did not have any functional changes but I cleaned its block diagram (hence the differences in the case structure sizes, nodes positions and overall wiring. I can remove it if you prefer.
